gui/launcher.h

```diff
@@ -23,14 +23,42 @@
 #ifndef GUI_LAUNCHER_DIALOG_H
 #define GUI_LAUNCHER_DIALOG_H
 
+// Disable the grid for platforms that disable fancy themes
+#ifdef DISABLE_FANCY_THEMES
+#define DISABLE_LAUNCHERDISPLAY_GRID
+#endif
+#define kSwitchLauncherDialog -2
+
 #include "gui/dialog.h"
 #include "engines/game.h"
 
+using Common::String;
```

Comment:
Please don't apply using in a header like this, use fully qualified names instead. (Or at least don't expand upon this, as I note we already did similar typedefs below)